featuretools/computational_backends/calculate_feature_matrix.py

```diff
@@ -759,7 +767,27 @@ def update_progress_callback_parameters(progress_bar, previous_progress):
 
 
 def init_ww_and_concat_fm(feature_matrix, ww_init_kwargs):
+    cols_to_check = {col for col, ltype in
+                     ww_init_kwargs["logical_types"].items()
+                     if isinstance(ltype, (Age, Boolean, Integer))}
+    replacement_type = {
+        "age": AgeNullable(),
+        "boolean": BooleanNullable(),
+        "integer": IntegerNullable(),
```

Comment:
Could we use the logical type strings for the values as well here?

Reply:
@tamargrey  Yeah, I think we could use the strings instead without issue, but I did it this way to be consistent with the other values in the logical types dictionary which are all `LogicalType` instances. I'm not sure that's important though. Do you have a preference here, or is there an advantage to using the strings other than avoiding an import?

Reply:
was just thinking to avoid the import, but I think keeping the types consistent with the rest of the dictionary is a fine reason to use the objects here. It'd make our lives easier if we ever try using this dictionary for anything further